### ISSUE TITLE
"h is not defined" when clicking "Add a screenshot

### ISSUE THREAD

**Issue by Nubebuster** [getsentry/sentry#73828]
### Environment

SaaS (https://sentry.io/)

### Steps to Reproduce

1. install sentry like so
```
import * as Sentry from "@sentry/react";

Sentry.init({
  dsn: "https://584ef4df9179d6209949fdbb8b49b688@o4507543529127936.ingest.de.sentry.io/4507543530897488",
  integrations: [
    Sentry.browserTracingIntegration(),
    Sentry.replayIntegration(),
    Sentry.feedbackIntegration({
      showBranding: false,
      colorScheme: "light",
    }),
  ],
  tracesSampleRate: 1.0,
  tracePropagationTargets: ["localhost", /^https:\/\/domain\.nl\/api/],
  replaysSessionSampleRate: 0.1,
  replaysOnErrorSampleRate: 1.0,
});
```
Click "Report a bug"
Click "Add a screenshot"

### Expected Result

Opens screenshot dialog

### Actual Result

Uncaught (in promise) ReferenceError: h is not defined
    df https://www.domain.nl/assets/index-7da3ba37.js:1328
    MZ https://www.domain.nl/assets/index-7da3ba37.js:983
    b_ https://www.domain.nl/assets/index-7da3ba37.js:983
    zP https://www.domain.nl/assets/index-7da3ba37.js:983
    LZ https://www.domain.nl/assets/index-7da3ba37.js:983
    b_ https://www.domain.nl/assets/index-7da3ba37.js:983
    zP https://www.domain.nl/assets/index-7da3ba37.js:983
    LZ https://www.domain.nl/assets/index-7da3ba37.js:983
    b_ https://www.domain.nl/assets/index-7da3ba37.js:983
    zP https://www.domain.nl/assets/index-7da3ba37.js:983
    LZ https://www.domain.nl/assets/index-7da3ba37.js:983
    b_ https://www.domain.nl/assets/index-7da3ba37.js:983
    zP https://www.domain.nl/assets/index-7da3ba37.js:983
    b_ https://www.domain.nl/assets/index-7da3ba37.js:983
    zP https://www.domain.nl/assets/index-7da3ba37.js:983
    LZ https://www.domain.nl/assets/index-7da3ba37.js:983
    b_ https://www.domain.nl/assets/index-7da3ba37.js:983
    zP https://www.domain.nl/assets/index-7da3ba37.js:983
    b_ https://www.domain.nl/assets/index-7da3ba37.js:983
    AZ https://www.domain.nl/assets/index-7da3ba37.js:983
    lp https://www.domain.nl/assets/index-7da3ba37.js:983
    promise callback*Ax https://www.domain.nl/assets/index-7da3ba37.js:983
    setState https://www.domain.nl/assets/index-7da3ba37.js:983
    __ https://www.domain.nl/assets/index-7da3ba37.js:983
    onClick https://www.domain.nl/assets/index-7da3ba37.js:983
    Nx https://www.domain.nl/assets/index-7da3ba37.js:983
    r https://www.domain.nl/assets/index-7da3ba37.js:889
    cQ https://www.domain.nl/assets/index-7da3ba37.js:892
    xG https://www.domain.nl/assets/index-7da3ba37.js:892
    cf https://www.domain.nl/assets/index-7da3ba37.js:983
    LZ https://www.domain.nl/assets/index-7da3ba37.js:983
    b_ https://www.domain.nl/assets/index-7da3ba37.js:983
    zP https://www.domain.nl/assets/index-7da3ba37.js:983
    LZ https://www.domain.nl/assets/index-7da3ba37.js:983
    b_ https://www.domain.nl/assets/index-7da3ba37.js:983
    zP https://www.domain.nl/assets/index-7da3ba37.js:983
    LZ https://www.domain.nl/assets/index-7da3ba37.js:983
    b_ https://www.domain.nl/assets/index-7da3ba37.js:983
    zP https://www.domain.nl/assets/index-7da3ba37.js:983
    LZ https://www.domain.nl/assets/index-7da3ba37.js:983
    b_ https://www.domain.nl/assets/index-7da3ba37.js:983
    zP https://www.domain.nl/assets/index-7da3ba37.js:983
    LZ https://www.domain.nl/assets/index-7da3ba37.js:983
    b_ https://www.domain.nl/assets/index-7da3ba37.js:983
    zP https://www.domain.nl/assets/index-7da3ba37.js:983
    b_ https://www.domain.nl/assets/index-7da3ba37.js:983
    zP https://www.domain.nl/assets/index-7da3ba37.js:983
    LZ https://www.domain.nl/assets/index-7da3ba37.js:983
    b_ https://www.domain.nl/assets/index-7da3ba37.js:983
    zP https://www.domain.nl/assets/index-7da3ba37.js:983
    LZ https://www.domain.nl/assets/index-7da3ba37.js:983
    b_ https://www.domain.nl/assets/index-7da3ba37.js:983
    zP https://www.domain.nl/assets/index-7da3ba37.js:983
    LZ https://www.domain.nl/assets/index-7da3ba37.js:983
    b_ https://www.domain.nl/assets/index-7da3ba37.js:983
    zP https://www.domain.nl/assets/index-7da3ba37.js:983
    b_ https://www.domain.nl/assets/index-7da3ba37.js:983
    zP https://www.domain.nl/assets/index-7da3ba37.js:983
    b_ https://www.domain.nl/assets/index-7da3ba37.js:983
    DZ https://www.domain.nl/assets/index-7da3ba37.js:983
    d https://www.domain.nl/assets/index-7da3ba37.js:1253
    open https://www.domain.nl/assets/index-7da3ba37.js:1253
    qe https://www.domain.nl/assets/index-7da3ba37.js:983
    r https://www.domain.nl/assets/index-7da3ba37.js:889
    cQ https://www.domain.nl/assets/index-7da3ba37.js:892
    xG https://www.domain.nl/assets/index-7da3ba37.js:892
    me https://www.domain.nl/assets/index-7da3ba37.js:983
    rt https://www.domain.nl/assets/index-7da3ba37.js:983
    setupOnce https://www.domain.nl/assets/index-7da3ba37.js:983
    EA https://www.domain.nl/assets/index-7da3ba37.js:880
    PV https://www.domain.nl/assets/index-7da3ba37.js:880
    PV https://www.domain.nl/assets/index-7da3ba37.js:880
    _setupIntegrations https://www.domain.nl/assets/index-7da3ba37.js:880
    init https://www.domain.nl/assets/index-7da3ba37.js:880
    DV https://www.domain.nl/assets/index-7da3ba37.js:881
    OQ https://www.domain.nl/assets/index-7da3ba37.js:892
    Iee https://www.domain.nl/assets/index-7da3ba37.js:1328
    <anonymous> https://www.domain.nl/assets/index-7da3ba37.js:1328
index-7da3ba37.js:1328:5564
    __e https://www.domain.nl/assets/index-7da3ba37.js:983
    b_ https://www.domain.nl/assets/index-7da3ba37.js:983
    AZ https://www.domain.nl/assets/index-7da3ba37.js:983
    lp https://www.domain.nl/assets/index-7da3ba37.js:983
    (Async: promise callback)
    Ax https://www.domain.nl/assets/index-7da3ba37.js:983
    setState https://www.domain.nl/assets/index-7da3ba37.js:983
    __ https://www.domain.nl/assets/index-7da3ba37.js:983
    onClick https://www.domain.nl/assets/index-7da3ba37.js:983
    Nx https://www.domain.nl/assets/index-7da3ba37.js:983
    r https://www.domain.nl/assets/index-7da3ba37.js:889
    (Async: EventListener.handleEvent)
    cQ https://www.domain.nl/assets/index-7da3ba37.js:892
    xG https://www.domain.nl/assets/index-7da3ba37.js:892
    cf https://www.domain.nl/assets/index-7da3ba37.js:983
    LZ https://www.domain.nl/assets/index-7da3ba37.js:983
    b_ https://www.domain.nl/assets/index-7da3ba37.js:983
    zP https://www.domain.nl/assets/index-7da3ba37.js:983
    LZ https://www.domain.nl/assets/index-7da3ba37.js:983
    b_ https://www.domain.nl/assets/index-7da3ba37.js:983
    zP https://www.domain.nl/assets/index-7da3ba37.js:983
    LZ https://www.domain.nl/assets/index-7da3ba37.js:983
    b_ https://www.domain.nl/assets/index-7da3ba37.js:983
    zP https://www.domain.nl/assets/index-7da3ba37.js:983
    LZ https://www.domain.nl/assets/index-7da3ba37.js:983
    b_ https://www.domain.nl/assets/index-7da3ba37.js:983
    zP https://www.domain.nl/assets/index-7da3ba37.js:983
    LZ https://www.domain.nl/assets/index-7da3ba37.js:983
    b_ https://www.domain.nl/assets/index-7da3ba37.js:983
    zP https://www.domain.nl/assets/index-7da3ba37.js:983
    b_ https://www.domain.nl/assets/index-7da3ba37.js:983
    zP https://www.domain.nl/assets/index-7da3ba37.js:983
    LZ https://www.domain.nl/assets/index-7da3ba37.js:983
    b_ https://www.domain.nl/assets/index-7da3ba37.js:983
    zP https://www.domain.nl/assets/index-7da3ba37.js:983
    LZ https://www.domain.nl/assets/index-7da3ba37.js:983
    b_ https://www.domain.nl/assets/index-7da3ba37.js:983
    zP https://www.domain.nl/assets/index-7da3ba37.js:983
    LZ https://www.domain.nl/assets/index-7da3ba37.js:983
    b_ https://www.domain.nl/assets/index-7da3ba37.js:983
    zP https://www.domain.nl/assets/index-7da3ba37.js:983
    b_ https://www.domain.nl/assets/index-7da3ba37.js:983
    zP https://www.domain.nl/assets/index-7da3ba37.js:983
    b_ https://www.domain.nl/assets/index-7da3ba37.js:983
    DZ https://www.domain.nl/assets/index-7da3ba37.js:983
    d https://www.domain.nl/assets/index-7da3ba37.js:1253
    open https://www.domain.nl/assets/index-7da3ba37.js:1253
    qe https://www.domain.nl/assets/index-7da3ba37.js:983
    InterpretGeneratorResume self-hosted:1412
    AsyncFunctionNext self-hosted:799
    (Async: async)
    r https://www.domain.nl/assets/index-7da3ba37.js:889
    (Async: EventListener.handleEvent)
    cQ https://www.domain.nl/assets/index-7da3ba37.js:892
    xG https://www.domain.nl/assets/index-7da3ba37.js:892
    me https://www.domain.nl/assets/index-7da3ba37.js:983
    rt https://www.domain.nl/assets/index-7da3ba37.js:983
    setupOnce https://www.domain.nl/assets/index-7da3ba37.js:983
    EA https://www.domain.nl/assets/index-7da3ba37.js:880
    PV https://www.domain.nl/assets/index-7da3ba37.js:880
    forEach self-hosted:157
    PV https://www.domain.nl/assets/index-7da3ba37.js:880
    _setupIntegrations https://www.domain.nl/assets/index-7da3ba37.js:880
    init https://www.domain.nl/assets/index-7da3ba37.js:880
    DV https://www.domain.nl/assets/index-7da3ba37.js:881
    OQ https://www.domain.nl/assets/index-7da3ba37.js:892
    Iee https://www.domain.nl/assets/index-7da3ba37.js:1328
    <anonymous> https://www.domain.nl/assets/index-7da3ba37.js:1328

### Product Area

User Feedback

### Link

https://www.meetingaanzee.nl/en/

### DSN

https://584ef4df9179d6209949fdbb8b49b688@o4507543529127936.ingest.de.sentry.io/4507543530897488

### Version

8.14.0

**Comment by gyannj** [comment]
I have the same issue, is there any fix?

**Comment by TheLiberal** [comment]
Getting the same, followed the setup from the wizard

**Comment by enszrlu** [comment]
same error here.

**Comment by ryan953** [comment]
Related to https://github.com/getsentry/sentry-javascript/pull/12784

**Comment by c298lee** [comment]
We are working on it, see https://github.com/getsentry/sentry-javascript/pull/12784 for a status. Alternatively, you can use an older SDK version eg. 8.12.0 until the fix is out.

**Comment by ryan953** [comment]
Related to https://github.com/getsentry/sentry-javascript/issues/12713

**Comment by ryan953** [comment]
This should be resolved and out in 8.16.0!

https://github.com/getsentry/sentry-javascript/issues/12713#issuecomment-2218479986
